libnetwork/datastore/datastore.go

```diff
@@ -102,7 +99,7 @@ func Key(key ...string) string {
 
 // New creates a new Store instance.
 func New(cfg Config) (*Store, error) {
-	s, err := boltdb.New(cfg.Path, cfg.Bucket, cfg.ConnectionTimeout)
```

Comment:
This was just added in the previous commit, correct? Perhaps amend that commit 🤔